kubemarine/resources/configurations/globals.yaml

```diff
@@ -52,7 +52,7 @@ pods:
       retries: 30
     plugins:
       timeout: 5
-      retries: 30
+      retries: 360
```

Comment:
Just a point to think of. Do we expect different time to become ready for different OOB plugins? If yes, we could specify timeout or retries in the defaults.yaml for the `expect` installation procedure directly, for example:
```yaml
        - expect:
            pods:
              list:
                - coredns
                - calico-kube-controllers
                - calico-node
              retries: 360
```

Reply:
Also I'd like to warn that unsuccessful installation will become not friendly, as it will hang the Kubemarine run for half an hour.